azure/packages/azure-local-service/README.md

```diff
@@ -45,11 +53,59 @@ $env:PORT=6502
 npm run start
 ```
 
-<!-- AUTO-GENERATED-CONTENT:START (README_CONTRIBUTION_GUIDELINES_SECTION:includeHeading=TRUE) -->
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README_FOOTER) -->
 
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
+## API Documentation
+
+API documentation for **@fluidframework/azure-local-service** is available at <https://fluidframework.com/docs/apis/azure-local-service>.
```

Comment:
This link 404s 🤔

Reply:
Yeah, I noticed that too. But there _should_ be docs for that.
@zhenmichael any idea why these API docs don't appear to be getting published to the website?